test/puppetlabs/trapperkeeper/bootstrap_test.clj

```diff
@@ -180,3 +269,24 @@ puppetlabs.trapperkeeper.examples.bootstrapping.test-services/foo-test-service ;
       ;; just test that this bootstrap config file can be read successfully
       ;; (ie, this does not throw an exception)
       (bootstrap-with-empty-config ["--bootstrap-config" bootstrap-url]))))
+
+(deftest duplicate-service-definitions
```

Comment:
Ah, yeah, sorry. I'd been working on that other ticket and forgot to switch branches before switching back to this ticket and did a git add -p. Missed this one.